app/src/main/res/layout/item_setting_account.xml

```diff
@@ -0,0 +1,108 @@
+    android:animateLayoutChanges="true"
+    android:orientation="vertical">
+
+    <com.google.android.material.card.MaterialCardView
+        android:id="@+id/accountCardview"
+        style="@style/CardViewMail"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:cardCornerRadius="@dimen/materialCardviewRadius">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:padding="@dimen/marginStandardMedium">
+
+            <com.google.android.material.card.MaterialCardView
+                android:id="@+id/userAvatar"
+                android:layout_width="38dp"
+                android:layout_height="0dp"
+                app:cardCornerRadius="100dp"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintDimensionRatio="1"
+                app:layout_constraintEnd_toStartOf="@id/userName"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent">
+
+                <ImageView
+                    android:id="@+id/userAvatarImage"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:contentDescription="TODO"
+                    android:src="@drawable/placeholder"
+                    tools:src="@tools:sample/avatars" />
+
+            </com.google.android.material.card.MaterialCardView>
+
+            <TextView
+                android:id="@+id/userName"
+                style="@style/H3"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/marginStandardMedium"
+                app:layout_constraintBottom_toTopOf="@id/userMailAddress"
+                app:layout_constraintEnd_toStartOf="@id/chevron"
+                app:layout_constraintStart_toEndOf="@id/userAvatar"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:text="Myriam Rais" />
+
+            <TextView
+                android:id="@+id/userMailAddress"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/marginStandardMedium"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toStartOf="@id/chevron"
+                app:layout_constraintStart_toEndOf="@id/userAvatar"
+                app:layout_constraintTop_toBottomOf="@id/userName"
+                tools:text="myriam.rais@infomaniak.com" />
+
+            <ImageView
+                android:id="@+id/chevron"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:rotation="-180"
```

Comment:
-180 is already used in [Extensions.kt](https://github.com/Infomaniak/android-mail/pull/132/files#diff-e3157e437107d6fb3048b238af15e5f383cba2ba90d28cf0ed1e9d98db7ea9f4) then you can use a `dimens` for this